app/javascript/components/mentoring/mentor_solutions_list/solution.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import dayjs from 'dayjs'
+import RelativeTime from 'dayjs/plugin/relativeTime'
+dayjs.extend(RelativeTime)
```

Comment:
You'll want to extract this to a new utils file so that you may replace the library with something else later. I recommend:

```javascript
// time.js

import dayjs from 'dayjs'
import RelativeTime from 'dayjs/plugin/relativeTime'
dayjs.extend(RelativeTime)

export { dayjs }
```

It would be even easier to replace, if you don't expose the entire export, and instead do:

```javascript
// time.js

import dayjs from 'dayjs'
import RelativeTime from 'dayjs/plugin/relativeTime'
dayjs.extend(RelativeTime)

export function fromNow(dateTime) {
  return dayjs(updatedAt).fromNow()
}
```

Reply:
Sounds good! Will open a PR for this.